src/utils/helpers.utils.js

```diff
@@ -107,3 +107,24 @@ export const checkUser = async (req, res, email) => {
   }
   return user;
 };
+
+export const isValidUser = async (req, res, next) => {
+  try {
+    const decodedToken = await verifyToken(
+      req.headers['x-access-token'] || req.headers.authorization,
+    );
+    req.userid = decodedToken.id;
+    return next();
+  } catch (err) {
+    console.log(err);
+    return errorResponse(res, 401, 'unauthorized users');
+  }
+};
+
+export const isArticleExist = async articleId => {
+  const getArticle = await Article.findOne({
+    where: { articleId: req.params.article_id },
+  });
+  if (getArticle) return getArticle;
+  return errorResponse(res, 404, 'article does not exist');
```

Comment:
How are you passing the `res` object to this function?

Reply:
the object uses the `res` object to form the error message and send it.
without `res`, `errorResponse()` and `successResponse()` would not be able to send the response to the client.